google-ads-examples/src/main/java/com/google/ads/googleads/examples/targeting/AddCustomerNegativeCriteria.java

```diff
@@ -0,0 +1,138 @@
+import com.google.ads.googleads.v1.services.MutateCustomerNegativeCriteriaResult;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This example adds various types of negative criterion for exclusions at the customer level. These
+ * criteria will be applied to all campaigns for the customer.
+ */
+public class AddCustomerNegativeCriteria {
+
+  private static class AddCustomerNegativeCriteriaParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private long customerId;
+  }
+
+  public static void main(String[] args) {
+    AddCustomerNegativeCriteriaParams params = new AddCustomerNegativeCriteriaParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new AddCustomerNegativeCriteria().runExample(googleAdsClient, params.customerId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Creates customer negative criteria
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  public static void runExample(GoogleAdsClient googleAdsClient, long customerId) {
+
+    try (CustomerNegativeCriterionServiceClient customerNegativeCriterionServiceClient =
+        googleAdsClient.getLatestVersion().createCustomerNegativeCriterionServiceClient()) {
+
+      // Create a negative customer criterion targeting the content label type of 'TRAGEDY'
```

Comment:
Instead of "targeting" on 100 and 106, please say "excluding" since these are negative criteria.

Reply:
Done.